x-pack/platform/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -1735,5 +1735,24 @@ export default function ({ getService }: FtrProviderContext) {
         expect(scheduledTask.enabled).to.be(false);
       });
     });
+
+    it('should update the retryAt of a long running task to be now + 5m', async () => {
+      const task = await scheduleTask({
+        taskType: 'sampleLongRunningRecurringTask',
+        schedule: { interval: `1d` },
+        params: {},
+      });
+
+      const now = Date.now();
+
+      await retry.try(async () => {
+        const scheduledTask = await currentTask(task.id);
+        const retryAt = Date.parse(scheduledTask.retryAt!);
+        expect(isNaN(retryAt)).to.be(false);
+
+        expect(retryAt).to.be.greaterThan(now + 5 * 60 * 1000);
+        expect(retryAt).to.be.lessThan(now + 6.5 * 60 * 1000);
```

Comment:
i guess it wouldn't be good to wait a minute for this test to update the retryAt again 🙈

Reply:
We can if you'd like me to! I am not sure the best way to test multiple updates

Reply:
It would be nice to be able to test that the `retryAt` is updated but not sure it's worth such a long running test.

Reply:
I can make a follow up issue to try and figure out a good test for this